### PR TITLE
SAK-45601: Modify "ifs" processing for XAPI events and pool usage instead of threads

### PR DESCRIPTION
**Pull request by victorGomollon**
related [https://github.com/sakaiproject/sakai/pull/9353](url)

### CODE REVIEW

File: docs/conversion/sakai_21_0-21_1_mysql_conversion.sql [hunk 1]
```diff
@@ -9,3 +9,36 @@ ALTER TABLE rbc_rating MODIFY title varchar(255) NOT NULL;
 UPDATE rbc_rating SET points = 0 WHERE points IS NULL;
 ALTER TABLE rbc_rating MODIFY points double NOT NULL;
 -- END SAK-45137
+
+-- START SAK-45601
+CREATE TABLE TINCANAPI_EVENT (
+EVENT VARCHAR(255) NOT NULL,
```
Comment (ottenhoff) [inline docs/conversion/sakai_21_0-21_1_mysql_conversion.sql:15]: Does this need to be 255 chars?

Comment (ottenhoff) [reply docs/conversion/sakai_21_0-21_1_mysql_conversion.sql:15]: Should use as small as possible to make life easier for indexes (e.g., utf8mb4 is 4 bytes per char = 255*4

Comment (victorGomollon) [reply docs/conversion/sakai_21_0-21_1_mysql_conversion.sql:15]: No problem, do you want a specific value? I think 64 would be enough. Does it seem right to you?

Comment (ottenhoff) [reply docs/conversion/sakai_21_0-21_1_mysql_conversion.sql:15]: Is it the same as SAKAI_EVENT->EVENT? That is varchar(32). Or is this different? Then 64 seems fine

Comment (victorGomollon) [reply docs/conversion/sakai_21_0-21_1_mysql_conversion.sql]: Done

**Pull request merged**